src/ui/components/pages/swap/hooks/useSwappableAssetTickersByVendor.ts

```diff
@@ -0,0 +1,46 @@
+import { useMemo } from 'react';
+import { SwapVendor } from 'swap/constants';
+import { useAppSelector } from 'ui/store';
+import { assetIds } from 'assets/constants';
+import { NetworkName } from 'networks/types';
+
+export type SwappableAssetIdRecord = Record<NetworkName.Mainnet, string[]>;
```

Comment:
Maybe rename it to `SwappableAssetIdsByNetwork` or something else?

Reply:
Agree